book/src/user/docker.md

```diff
@@ -54,10 +52,34 @@ docker build \
   .
 ```
 
+### Alternatives
+
+See [Building Zebra](https://github.com/ZcashFoundation/zebra#building-zebra) for more information.
+
+## Advanced usage
+
+You're able to specify various parameters when building or launching the Docker image, which are meant to be used by developers and CI pipelines. For example, specifying the Network where Zebra will run (Mainnet, Testnet, etc), or enabling features like metrics with Prometheus.
```

Comment:
I'd say selecting between Mainnet and Testnet and monitoring is basic functionality for all users. I'd rather avoid orienting it at advanced users or devs.

```suggestion
```